consensus/src/pow/work.rs

```diff
@@ -106,15 +106,15 @@ impl MiningResult {
     }
 }
 
-pub fn calculate_work_required<G>(
+pub fn calculate_work_required<F>(
     chain_config: &ChainConfig,
     prev_block_index: &GenBlockIndex,
     block_timestamp: BlockTimestamp,
     pow_status: &PoWStatus,
-    get_ancestor: G,
+    get_ancestor: F,
```

Comment:
Is there a reason you're naming this from F to G? Btw it's G for Get or Getter.

Reply:
I faintly remember there were originally two functions - F and G, and F was removed. If not for this function, for another one, and so I was just "resetting the counter" so to speak. But you're rith - G for Getter works too, so I'll leave as is and drop that commit.